email/1.4/modules/ROOT/pages/email-troubleshooting.adoc

```diff
@@ -0,0 +1,196 @@
+= Troubleshoot Email Connector - Mule 4
+
+To troubleshoot Anypoint Connector for Email (Email Connector), become familiar with the information about app logs, JavaMail debugging, troubleshooting access attachments, troubleshooting protocols issues, and interpreting commonly thrown messages.
+
+== View the App Log
+
+If you encounter problems while running your Mule runtime engine (Mule) app, view the app log as follows:
+
+* If you’re running the app from Anypoint Platform, the output is visible in the Anypoint Studio console window.
+* If you’re running the app using Mule from the command line, the app log is visible in your OS console.
+
+Unless the log file path is customized in the app’s log file `log4j2.xml`, view the app log in the default location `MULE_HOME/logs/<app-name>.log`.
+
+
+== Enable JavaMail Debug Logging
+
+You can enable JavaMail debugging logging which is used by the underlying third-party classes.
+
+To enable JavaMail debugging logging:
+
+. Access Anypoint Studio and navigate to the *Package Explorer* view.
+. Open your application's project name.
+. Open the `src/main/mule` path folder.
+. Open your Mule app XML file inside the folder.
+. Add the `<property key="mail.debug" value="true"/>` on the IMAP or SMTP connection element.
++
+[source,xml,linenums]
+----
+  <email:imaps-connection host="..." user="..." password="..." >
+    <email:properties >
+      <email:property key="mail.debug" value="true" />
+    </email:properties>
+  </email:imaps-connection>
+----
++
+[start=6]
+. Save your changes.
+. Click the project name in *Package Explorer* and then click *Run* > *Run As* > *Mule Application*.
+. Navigate to the *Console* view to read the Email messages in the logger:
++
+[source,plain-text]
+----
+DEBUG: JavaMail version 1.6.3
+DEBUG: successfully loaded resource: /META-INF/javamail.default.address.map
+DEBUG: getProvider() returning javax.mail.Provider[STORE,imap,com.sun.mail.imap.IMAPStore,Oracle]
+DEBUG IMAP: mail.imap.fetchsize: 16384
+DEBUG IMAP: mail.imap.ignorebodystructuresize: false
+DEBUG IMAP: trying to connect to host "******", port 993, isSSL true
+* OK The Microsoft Exchange IMAP4 service is ready. [******]
+A0 CAPABILITY
+* CAPABILITY IMAP4 IMAP4rev1 AUTH=PLAIN AUTH=XOAUTH2 SASL-IR UIDPLUS MOVE ID UNSELECT CHILDREN IDLE NAMESPACE LITERAL+
+A0 OK CAPABILITY completed.
+...
+----
+
+If you run the app in Anypoint Studio, the logging prints the debug lines in the console view. +
+If you run the app in a standalone Mule, the logging prints the debug lines in the log file.
+
+
+== Enable Email Connector Debug Logging
+
+To begin troubleshooting Email Connector, enable debug logging to see the exact error messages and its trace:
```

Comment:
To what does "its" in "its trace" refer?
If the connector, clarify; if the "error messages," then there is numerical disagreement between "messages" (plural) and "its trace" (singular).